.github/workflows/horus.yml

```
@@ -19,11 +19,13 @@ jobs:
     - name: Get the list of packages
       run: |
         cp jammy/packages_to_import /tmp/patched-packages
-    - name: Install Dependencies
+    - name: Install Dependencies and enable Sources
```

```suggestion
    - name: Install dependencies and enable sources
```

Let's match the case style while we're here.